shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/communication/jdbc/recognizer/impl/H2Recognizer.java

```diff
@@ -27,9 +28,11 @@
  */
 public final class H2Recognizer implements JDBCDriverURLRecognizer {
     
+    private final H2DatabaseType databaseType = new H2DatabaseType();
+    
     @Override
     public String getDatabaseType() {
```

Comment:
Can this method revert to origin?

Reply:
I saw the code contains so many literals of databaseType,  is it better to refers only *DatabaseType class ?